### DISCUSSION TITLE
Customer with older active accounts

### DISCUSSION THREAD

**Discussion by Legeakademiet** [opengento/magento2-gdpr#142] [Q&A] [answered]
First of all thanks for a promising GDPR module :)

I'm wondering what will happend to a customer account that was created 5 years ago but the account is still active when I have set the "Entities Lifetime" to 2 years. Will the account be Anonymized in this case after 2 years?

**Comment by thomas-kl1** [comment, marked as answer]
Yes, the label is misleading and should be updated to be more meaningful. The anonimyzation start after 2 years of inactivities (based on the updated_at value).

**Reply by Legeakademiet** [reply]
Thank you for replying. :)
